specs.json

```
@@ -612,7 +612,7 @@
   "https://www.w3.org/TR/web-share/",
   "https://www.w3.org/TR/webaudio/",
   {
-    "url": "https://www.w3.org/TR/webauthn-2/",
+    "url": "https://www.w3.org/TR/webauthn-333/",
```

One `3` might be enough...

```suggestion
    "url": "https://www.w3.org/TR/webauthn-3/",
```

---

that's called being ahead of the game!